.github/workflows/test-graphql.yml

```diff
@@ -0,0 +1,29 @@
+name: Test GraphQL generation
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+    paths:
+      - '.github/workflows/test-graphql.yml'
+      - 'packages/eas-cli/src/graphql/**'
+  schedule:
+    - cron: '0 8 * * *' # 8:00 AM UTC time every day
```

Comment:
I think GraphQL API in `www` is changing rather often and we're going to get a lot of failures here.

Maybe instead we should make some renovatebot thing update GraphQL on its own and submit pull requests?